tests/features/capsulahub-application.feature

```diff
@@ -0,0 +1,80 @@
+
+ Feature: CapsulaHub application
+
+ Background:
+    Given a npm package called @capsulajs/capsula-hub
+    And   a default port 55555
+    And   a valid port is a number between 1024 and 65535
+    And   two existing config files conf.js and conf1.js located in capsulahub/src/config
```

Comment:
Configuration of the Workspace is an external file and should not exist inside **capsulahub** source code.